static/js/index.js

```diff
@@ -4,11 +4,20 @@ import {
 
 import showPopUp from './popUp.js';
 
+import {
+  removeErrorMessage,
+  validateEmail,
+} from './validators.js';
+
 window.addEventListener('resize', () => { window.location.reload(); });
 if (window.innerWidth < 992) {
   addEvents();
 }
 
 document.querySelectorAll('.see_more').forEach((button) => {
   button.addEventListener('click', () => showPopUp(button.dataset.id));
-});
\ No newline at end of file
+});
+
+document.querySelector('#contact-form').addEventListener('submit', validateEmail);
+
+document.querySelector('input[name="email"]').addEventListener('input', removeErrorMessage);
```

Comment:
[OPTIONAL] I think we could wrap all of our JS code in a `main` function. And the call `main` once the web app is finished loading. This will ensure that it works correctly. Imagine that the script loads, but maybe some of the elements on the document aren't still ready. Then it could fail to select these.

Start the script running once the page has finished loading like this: `window.onload = main`